install.js

```diff
@@ -304,6 +305,14 @@ function requestBinary(requestOptions, filePath) {
   return deferred.promise
 }
 
+function getTargetPlatform(){
+  if (process.env.PHANTOMJS_PLATFORM){
```

Comment:
ok, let me ask around a bit and get some opinions from people that have dealt with cross-compilation.

as you point out, cross-compilation is a tricky beast...like what if one package needs phantomjs built for the host arch, and another package needs phantomjs built for the target arch? so i'd like to get other opinions before committing to support this (though i agree it seems like a nice thing to support).

for what it's worth, i believe most people handle your use case by running `npm install` on the host machine and then running `npm rebuild` on the target machine as part of the deploy process.